src/applications/appeals/996/tests/hlr-keyboard-only.cypress.spec.js

```diff
@@ -118,12 +118,27 @@ describe('Higher-Level Review keyboard only navigation', () => {
         chapters.informalConference.pages.representativeInfoV2.path,
       );
       const rep = data.informalConferenceRep;
-      const repPrefix = '#root_informalConferenceRep_';
-      cy.typeInIfDataExists(`${repPrefix}firstName`, rep.firstName);
-      cy.typeInIfDataExists(`${repPrefix}lastName`, rep.lastName);
-      cy.typeInIfDataExists(`${repPrefix}phone`, rep.phone);
-      cy.typeInIfDataExists(`${repPrefix}extension`, rep.extension);
-      cy.typeInIfDataExists(`${repPrefix}email`, rep.email);
+      const repPrefix = '[name="root_informalConferenceRep_';
+      cy.get(`${repPrefix}firstName"]`)
+        .shadow()
+        .find('input')
+        .type(rep.firstName);
```

Comment:
I'm not sure why these were changed, for keyboard-only, but it should still work as-is. The command `typeInIfDataExists` essentially does the following:

```js
cy.tabToElement('selector');
cy.typeInFocused('text');
```

Reply:
@Mottie an error was returning for `typeInIfDataExists` b/c it was finding 2 duplicate elements since the `VaTextInputField` has a shadow that introduced a second pair of the name and id.

Reply:
Then change the code to:

```js
const repPrefix = 'input#root_informalConferenceRep_';
cy.typeInIfDataExists(`${repPrefix}firstName`, rep.firstName);
cy.typeInIfDataExists(`${repPrefix}lastName`, rep.lastName);
cy.typeInIfDataExists(`${repPrefix}phone`, rep.phone);
cy.typeInIfDataExists(`${repPrefix}extension`, rep.extension);
cy.typeInIfDataExists(`${repPrefix}email`, rep.email);
```

Because using `get` and `find` defeats the purpose of this keyboard-only test. You're basically clicking on the input.

Reply:
I was pairing with Rocio on this yesterday — it still gave us an error about more than one element when we put added `input` to the selector and when we inspected the elements, the id was actually not `root_informalConferenceRep_` anymore, it was `inputField` (which is weird because I thought IDs had to be unique but the components that Rocio added all had `#inputField`). IIRC both the input element and the web component wrapping it had the same name and id, which is why we got more specific about adding `shadow`, but I might be misremembering.

I wonder if there's a middle ground where we use `tabToElement` to get to the web component and then use `shadow` and `find` to get the input and avoid the error? Idk

Reply:
Hmmm, I believe `inputField` is the ID added to the input inside the shadow DOM. So we should be targeting the unique name instead. I'm gonna pair with Rocio and look at this.